src/index.js

```diff
@@ -53,6 +54,9 @@ const defaultRuleProperties = {
   schemaJsonFilepath: {
     type: 'string',
   },
+  schemaString: {
```

Comment:
what do you think about using `typeDefs` instead? its a term i saw a lot in [graphql-tools](https://github.com/apollographql/graphql-tools) when referring to the schema when defined via the IDL